packages/venia-ui/lib/util/apolloCache.js

```diff
@@ -54,6 +55,9 @@ export const cacheKeyFromType = object => {
         // Only maintain a single cart entry
         case MagentoGraphQLTypes.Cart:
             return 'Cart';
+        // Only maintain single customer entry
+        case MagentoGraphQLTypes.Customer:
+            return MagentoGraphQLTypes.Customer;
```

Comment:
I'm actually unsure of the purpose of this Map, since all the keys are identical to the values. Might want to check with @supernova-at if that is still needed.

Either way, updated for you in b5aec84 😉

Reply:
Spike ticket :) https://jira.corp.magento.com/browse/PWA-613